File: CustomerReport/Model/CMROutputBean.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports IDAUtil

Public Class CMROutputBean

    <Column("[soldTo]")>
    Public Property soldTo As Long
    <Column("[soldToName]")>
    Public Property soldToName As String
    <Column("[shipTo]")>
    Public Property shipTo As Long
    <Column("[shipToName]")>
    Public Property shipToName As String
    <Column("[salesOrg]")>
    Public Property salesOrg As String
    <Column("[id]")>
    Public Property id As String

    Public Sub New(zv As ZV04HNProperty, id As String)
        soldTo = zv.soldto
        soldToName = zv.soldtoName
        shipTo = zv.shipto
        shipToName = zv.shiptoName
        Me.id = id
    End Sub
End Class

File: CustomerReport/Model/CMROutputBean.cs
using IDAUtil;
using IDAUtil.Model.Properties.TcodeProperty.ZV04Obj;
using System.ComponentModel.DataAnnotations.Schema;

namespace CustomerReport {
    public class CMROutputBean {
        [Column("[soldTo]")]
        public long soldTo { get; set; }
        [Column("[soldToName]")]
        public string soldToName { get; set; }
        [Column("[shipTo]")]
        public long shipTo { get; set; }
        [Column("[shipToName]")]
        public string shipToName { get; set; }
        [Column("[salesOrg]")]
        public string salesOrg { get; set; }
        [Column("[id]")]
        public string id { get; set; }

        public CMROutputBean(ZV04HNProperty zv, string id) {
            soldTo = zv.soldto;
            soldToName = zv.soldtoName;
            shipTo = zv.shipto;
            shipToName = zv.shiptoName;
            this.id = id;
        }
    }
}